app/src/main/java/com/example/themoviesdb/ui/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.themoviesdb.ui
+
+import android.os.Bundle
+import android.view.*
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.bumptech.glide.Glide
+import com.example.themoviesdb.R
+import com.example.themoviesdb.databinding.FragmentMovieDetailsBinding
+import com.example.themoviesdb.domain.model.Movie
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MovieDetailsFragment : Fragment() {
+
+    private lateinit var binding: FragmentMovieDetailsBinding
+    private val mainViewModel by viewModels<MainViewModel>()
```

Comment:
you are not utilizing this new instance of mainViewModel here other than assigning one variable, either use the same instance of mainViewModel from MovieListFragment, or just use the movie object passed from arguments